resources/windows/wix/project.wxs.erb

```diff
@@ -25,6 +25,7 @@
       <!-- We can add all components by referencing this one thing -->
       <ComponentGroupRef Id="ProductComponentGroup" />
       <ComponentGroupRef Id="RegistryComponentGroup" />
+      <ComponentRef Id="service_nssm" />
```

Comment:
As discussed in e-mail - suggest we add componentrefs.wxs.erb to pick these up from the services.